history.go

```diff
@@ -16,15 +19,16 @@ func WriteFile(filePath string, lines []string) error {
 	}
 	for _, line := range lines {
 		// It is complicating the tests
-		//f.WriteString(fmt.Sprintf("[%d] %s%s", time.Now().Unix(), line, "\n"))
 		f.WriteString(fmt.Sprintf("%s%s", line, "\n"))
+		// log.SetOutput(f)
```

Comment:
No commented-out code allowed at BIT! We can delete the code altogether, in the sure and certain knowledge that Git has it all saved in case we need it later. If we want to log debug information, we can accept a logger at our main entry point, and log to that. In the tests, we can inject a null logger (such as one constructed from `ioutil.Discard`), and in the real tool, we can inject one based on `os.Stderr`.

Reply:
sorry. I removed some but forgot this.